jdaviz/components/plugin_table.vue

```diff
@@ -54,9 +54,8 @@
     </v-row>
 
     <v-row v-if="clear_table" justify="end">
-      <v-btn
-        color="primary"
-        text
+      <plugin-action-button
+        :results_isolated_to_plugin="true"
         @click="clear_table"
         >Clear Table
       </v-btn>
```

Comment:
```suggestion
      </plugin-action-button>
```
I think you missed this one.

Reply:
woops!!